src/System.Private.ServiceModel/src/System/ServiceModel/Security/X509CertificateInitiatorClientCredential.cs

```diff
@@ -27,6 +27,14 @@ internal X509CertificateInitiatorClientCredential(X509CertificateInitiatorClient
             _isReadOnly = other._isReadOnly;
         }
 
+        internal bool CloneCertificate
+        {
+            get
+            {
+                return RuntimeInformation.IsOSPlatform(OSPlatform.Windows);
+            }
+        }
```

Comment:
Probably worth adding some comments to explain why this should return true if it's windows.